ci-operator/step-registry/optional-operators/cvp-common/apply-secrets/optional-operators-cvp-common-apply-secrets-commands.sh

```diff
@@ -9,14 +9,7 @@ PYXIS_URL="${PYXIS_URL:-""}"
 # The namespace into which the operator and catalog will be
 # installed. Special value `!create` means that a new namespace will be created.
 OO_INSTALL_NAMESPACE="${OO_INSTALL_NAMESPACE:-$REHEARSAL_INSTALL_NAMESPACE}"
-
-# Check if PYXIS_URL exists, skip the whole step if not.
-if [[ -z "$PYXIS_URL" ]]; then
-    echo "PYXIS_URL is not defined, skipping step cvp-common-apply-secrets!"
-    exit 0
-else
-    echo "PYXIS_URL is defined, proceeding with cvp-common-apply-secrets step."
-fi
+OO_PACKAGE="${OO_PACKAGE:-""}"
```

Comment:
Is this safe? What would happen if `OO_PACKAGE` was empty?

Reply:
Yes, 
OO_PACKAGE will never be empty because it will be either provided as a ENV variable or overridden by CVP trigger

Reply:
In rehearsal i can default it CPaaS test operator bundle. 
@apodhrad